Userland/Utilities/find.cpp

```diff
@@ -567,7 +537,31 @@ static void walk_tree(FileData& root_data, Command& command)
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    LexicalPath root_path(parse_options(arguments.argc, arguments.argv));
+    Vector<char*> args;
+    args.ensure_capacity(arguments.argc);
+    for (int i = 1; i < arguments.argc; ++i)
+        args.unchecked_append(arguments.argv[i]);
```

Comment:
Maybe
```c++
auto args = arguments.strings;
args.take_first();
```
Instead?

Reply:
Oh it's a bunch of `char*`s
Then how about
```c++
Vector<char*> args;
args.append(arguments.argv + 1, arguments.argc - 1);
```
?

Reply:
I was looking at what LibMain is doing (the code that is currently in the PR is basically a copy of that with changed bounds and without the Span conversion), but if what you suggest works that would be even better.